docs/content/overview/cdc-pipeline.md

```diff
@@ -0,0 +1,107 @@
+Each event is marked with an ID (called **Table ID**) for which the change occurred. Events are categorized into SchemaChangeEvents and DataChangeEvents, representing changes in table structure and data respectively.
+
+**Data Source** Connector captures the changes in storage objects in the upstream system and converts them into events to be output to the synchronization task. It also provides a MetadataAccessor for the framework to read the metadata of the upstream system.
+
+The connector for the data sink is called the **Data Sink**. Upon receiving an event, the Data Sink applies the changes to the storage objects in the target system, achieving data synchronization with the upstream system. Additionally, MetadataApplier is used to apply metadata changes from the upstream system to the target system.
+
+Since events flow from the upstream to the downstream in a pipeline manner, the data synchronization task is also referred to as a **Data Pipeline**. A Data Pipeline consists of a Data Source, Route, Transform and Data Sink. The transform can modify events to some extent, and the router can remap the table IDs corresponding to events.
+
+Next, let's provide a detailed introduction to the concepts you need to know when using the CDC pipeline.
+
+### Table ID
+When connecting to external systems, it is necessary to establish a mapping relationship with the storage objects of the external system. This is what Table Id refers to.
+
+To be compatible with most external systems, the Table ID is represented by a 3-tuple, (namespace, schemaName, table). Connectors need to establish the mapping between Table ID and storage objects in external systems. For instance, a database table in MySQL/Doris is mapped to (null, database, table), and a topic in a message queue system such as Kafka is mapped to (null, null, topic).
+
+### Data Source
+Data Source is used to access metadata and read the changed data from external systems.
+A Data Source can read data from multiple tables simultaneously.
+
+To describe a data source, the follows are required:
+* Type: The type of the source, such as MySQL, Postgres.
+* Name: The name of the source, which is user-defined (optional, with a default value provided).
+* Other custom configurations for the source.
+
+
+For examle, we can use yaml to define a mysql source:
+```yaml
+source:
+  type: mysql
+  name: mysql-source   #optional，description information
+  host: localhost
+  port: 3306
+  username: admin
+  password: pass
+  tables: adb.*, bdb.user_table_[0-9]+, [app|web]_order_.*
+```
+The ability to synchronize the entire library comes from the implementation of Flink source. After obtaining the user configuration, the Flink source connector needs to be able to send all data in a database downstream according to the specified data type.
+
+### Data Sink
+The Data Sink is used to modify metadata and write change data to external systems. A Data Sink can write to multiple tables simultaneously.
```

Comment:
The Data Sink is used to apply schema changes and write change data to external systems.